client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -46,26 +65,183 @@ const List = React.createClass( {
 					path={ this.props.context.path }
 					cart={ this.props.cart }
 					selectedSite={ this.props.selectedSite } />
-
 				{ this.domainWarnings() }
 
 				<SectionHeader label={ headerText }>
-					{ this.addDomainButton() }
+					{ this.headerButtons() }
 				</SectionHeader>
 
 				<div className="domain-management-list__items">
+					{ this.notice() }
 					{ this.listItems() }
 				</div>
 			</Main>
 		);
 	},
 
+	hideNotice() {
+		this.setState( { notice: null } );
+	},
+
+	notice() {
```

Comment:
I was thinking about doing that but couldn't think of an quick and easy way. Functional components might be the answer!